src/Mentat.UI/Models/MongoHelper.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using MongoDB.Bson;
+using MongoDB.Driver;
+using System.Configuration;
+
+
+namespace Mentat.UI.Models
+{
+    public class MongoHelper
+    {
+        public static IMongoClient client { get; set; }
+
+        public static IMongoDatabase database { get; set; }
+        public static string MongoConnection =  "mongodb+srv://cdb:eN9qAv7forAKSnEt@flashcardapp.4yggo3z.mongodb.net/?retryWrites=true&w=majority";
+        public static string MongoDatabase = "fcApp"; //name of Database Instance
+   
+        public static IMongoCollection<Models.Card> FlashCards { get; set; }
```

Comment:
Does adding a using statement let you get rid of having to write out the namespace?


using Mentat.UI.Models;